{{cookiecutter.package_name}}/.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.9, 3.10, 3.11]
```

Comment:
I'm pretty sure this is going to break if they're not in strings when you get to two decimals. Also, I think we still support 3.8?

`python-version: ["3.8", "3.9", "3.10", "3.11"]

Reply:
Interesting. Why does two decimals confuse it?